src/main/java/pl/psnc/dei/model/DAO/TranscriptionRepository.java

```diff
@@ -6,4 +6,5 @@
 
 @Repository
 public interface TranscriptionRepository extends JpaRepository<Transcription, Long> {
+    boolean existsByRecord_Identifier(String recordId);
```

Comment:
I think you should not use _ in the method name

Reply:
https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#repositories.query-methods.query-property-expressions